main.py

```diff
@@ -909,9 +913,9 @@ def render_view(resource, view=None, args=None):
             if 'as' in request.args:
                 types = [URIRef(request.args['as']), 0]
 
-            types.extend([(x, 1) for x in self.vocab[resource.identifier : RDF.type]])
+            types.extend([(x, 1) for x in self.vocab[resource.identifier : NS.RDF.type]])
```

Comment:
Another situation where the intermediate list is not necessary. .extend can take a generator, like:

.extend((x, 1) for x in self.vocab[resource.identifier : RDF.type])